src/main/java/core/basesyntax/db/StorageImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageImpl implements Storage {
+    private Map<String, Integer> fruitMap;
+
+    public StorageImpl() {
+        fruitMap = new HashMap<>();
+    }
+
+    @Override
+    public void add(String fruit, int quantity) {
+        fruitMap.put(fruit, quantity);
+    }
+
+    @Override
+    public void change(String fruit, int newQuantity) {
+        fruitMap.replace(fruit, newQuantity);
```

Comment:
The `change` method name is ambiguous. Consider renaming it to reflect its purpose more clearly, such as `updateQuantity` or `setQuantity`.